libs/sdk-ui-dashboard/src/presentation/dragAndDrop/draggableAttributeFilter/DraggableAttributeFilter.tsx

```diff
@@ -84,6 +96,7 @@ export function DraggableAttributeFilter({
             >
                 <FilterComponent
                     filter={filterToUse}
+                    workingFilter={enableDashboardFiltersApplyModes ? workingFilterToUse : undefined}
```

Comment:
Is this prop really required?

Reply:
After discussion on Slack we decided to deprecate this prop for now and remove it in the future phase 5 refactoring.

Reply:
We will pass workingFilterToUse into regular filter prop instead.

Which means the inner filter state will think those incoming working filters are applied. But we do not need to distinguish between applied and working state for now. And we prefer to keep filters component API simple.

Similar change should be done for DateFilter.